scipy-1.0/paper.tex

```diff
@@ -185,6 +186,114 @@ \section*{Project organisation and community}
 external representative on the NumFOCUS Board of Directors.
 
 \textbf{Roadmap}
+The SciPy Roadmap\cite{SciPy_roadmap} is a continuously-updated document
+maintained by the community that describes some of the major directions
+for improvement for the project, as well as specific limitations and
+matters that require assistance moving forward.
+
+In general, we encourage changes that improve clarity in the API of the library,
+but strongly discourage breaking backwards compatibility, given our position near
+the base of the scientific Python computing stack.
+
+Unit test coverage for most recent code additions / adjustments is quite
+respectable, with overall project test coverage holding steady at
+around 76\% for the last six months (Figure~\ref{fig:coverage}),
+but some of the historical components of the code base
+would still benefit from increased test coverage. We are still striving
+to increase the number of high-quality SciPy usage tutorials beyond
```

Comment:
probably should remove "high-quality", because that's not really what they are I think - they could use some work